ts/project.ts

```diff
@@ -247,6 +268,20 @@ const proxyEnvironmentHelpers = (): ProjectHelpers => ({
     return templateLiteralFn;
   },
 
+  addPackage<T extends Project, Name extends string>(
+    this: T,
+    name: Name,
+    pkg: string,
+  ) {
+    const build = this.defaultEnvironment
+      ? this.build(pkg)
+      : emptyEnvironment.build(pkg);
```

Comment:
For `Project.addCheck` and `Project.addExecutable` we decided that -- if there's no default environment -- we crash. (I wonder whether we should reconsider that.) But why is `Project.addPackage` different?

Reply:
Ah my bad, for some reason I thought we only had that restriction on `.build` and `.shell` but not on the `.add*` methods. Will update